docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -130,4 +130,16 @@ When you attach an indicator to a case, the indicator is added as a new comment
 To remove an indicator attached to a case, click *More actions* (*...​*) -> *Delete attachment* in the case comment. 
 
 [role="screenshot"]
-image::images/remove-indicator.png[Removing an indicator from a case]
\ No newline at end of file
+image::images/remove-indicator.png[Removing an indicator from a case]
+
+[discrete]
+[[add-indicator-to-blocklist]]
+== Add indicators to the blocklist
+
+Add indicators to the <<blocklist,blocklist>> to prevent selected applications from running on your hosts. You can add indicators that have the `file` indicator type and hash values for the MD5, SHA-1, or SHA-256 fields.
```

Comment:
How exactly do indicators translate to blocklist entries? How does an indicator specify "selected applications"? Does this mean that any program/file that has a hash in an indicator that you've added to the blocklist won't run? 

(More of a tangent, but I also wonder how users would decide to add a particular indicator to the blocklist, rather than a whole bunch/all of them.)

Reply:
Ok, sorry for taking a while respond to this :persevere: 

My very, very, _very_ high-level understanding (and @PhilippeOberti please correct me if I'm wrong about any of this) is that `file` type indicators with certain MD5, SHA-1, or SHA-256 hash values can be indicative of corrupted or malicious files that you don't want executed or downloaded in your environment. In [our instructions](https://www.elastic.co/guide/en/security/master/blocklist.html) for setting up a blocklist, we allude to this in Step 4b but it's not super explicit. In these instructions and the blocklist ones, we're likely assuming the that user has some knowledge of MD5, SHA-1, or SHA-256 hash values and knows which are IoCs, but adding some light background could help users who are less informed on the subject (like myself 😅).

Reply:
I fee like maybe the `Add indicators to the blocklist` could be a bit misleading here. We're not really adding indicators, we're creating a blocklist entry using a field from an IoC.
As you can see in the image below, the blocklist entry (created from an IoC) will look at the `file.hash` property with the value retrieved from the IoC at the time of creation.
![Screenshot 2023-03-28 at 4 23 51 PM](https://user-images.githubusercontent.com/17276605/228370342-0c1252d5-537b-42e1-a569-6acaf606120d.png)

Does this help or does it bring more confusion?

Reply:
@PhilippeOberti I think I get what you're saying. With this feature, users are expanding their blocklist to include certain information that they gathered from analyzing IoCs, not the full IoC itself. Is that correct? Also, are you recommending that I alter the section title and the phrasing? For example:

Use data from IoCs to expand the blocklist

Add indicator values to the [blocklist](https://security-docs_3024.docs-preview.app.elstc.co/guide/en/security/master/blocklist.html) to prevent selected applications from running on your hosts. You can use values from `file` type indicators that have MD5, SHA-1, or SHA-256 hash values.

You can add indicator values to the blocklist from the Indicators table or the Indicator details flyout. From the Indicators table, select More actions (…​​) → Add blocklist entry. Alternatively, open an indicator’s details, then select Take action → Add blocklist entry.

Reply:
yes and yes :)